portfolio/src/main/webapp/index.html

```diff
@@ -23,21 +25,24 @@ <h2>Nicole's Gallery</h2>
             </div>
         </div>
     </div>
-
-    Intro paragraph of who I am.
-      <p>Hi everyone! My name is Nicole and this is my portfolio! So a little about me I am a 3rd year CS major at Engineering school at the University of Virgina. I am originally from Northern Virginia (NOVA as we like to call it). 
+    
+    <!--About me section-->
+    About Me:
+      <p id="me">Hi everyone! My name is Nicole and this is my portfolio! So a little about me I am a 3rd year CS major at Engineering school at the University of Virgina. I am originally from Northern Virginia (NOVA as we like to call it). 
         I am a first gen college student as both of my parents are immigrants. My mom is from Peru and my dad is from Honduras. I am the eldest of two with just a younger sister who is going to go to the rival university of mine!
         Outside of school I really love to travel. So far I have been to 12 different countries and hopefully after I graduate I will be able to travel again. I also really love taking care of kids. I used to work as a swim instructor
         for a year and half teaching mommy and me's and toddlers. I also nannied last summer and would volunteer as at a local preschool in Charlottesville. I am also a HUGE Disney fan. My house is filled with Disney decorations, stuff animals, appliances 
         you name it. Right now I am just trying to get back into a normal routine. This school year has really thrown me for a loop so at the moment I am just trying to get back into working out, eating a normal times again(lol), and trying to take some summer
         classes along with SPS. So thats just a brief overview of who I am. Hopefully this portfolio will give you an insight to who I am. </p>
-      
+    </div>
```

Comment:
I think this should be `<div>` to match the closing tag at line 82.
```suggestion
    <div>
```